# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fget/errno.patch
# Copyright (C) 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -ur fget/error.h fget-fixed/error.h
--- fget/error.h	2000-07-01 23:36:17.000000000 +0200
+++ fget-fixed/error.h	2005-12-16 20:02:23.000000000 +0100
@@ -1,6 +1,8 @@
+
 #ifndef ERROR_H
 #define ERROR_H
 
+#include <errno.h>
 #ifndef errno
 extern int errno;
 #endif
